### DISCUSSION TITLE
Can accessing a mergerfs mount via a symlink, confuse it?

### DISCUSSION THREAD

**Discussion by dagelf** [trapexit/mergerfs#1194] [General]
I had a create policy of newest, which wasn't getting honored:

Steps to repeat:
````
/mnt/mergerfs
mkdir /mnt/underlying/a/folder
touch /mnt/mergerfs/folder/file  
mkdir /mnt/underlying/b/folder
ln -s /mnt/mergerfs /test
cd /test
rm folder/file
touch folder/file
# this file should be in b, right?
````
It wasn't in my case, but maybe I missed something. Need to get some sleep now. Will test later.

**Comment by trapexit** [comment]
Removing the file from /mnt/underlying/a/folder updates the mtime. So no, I wouldn't expect it to go to b.
